dask/dataframe/core.py

```diff
@@ -6405,28 +6405,19 @@ def get_parallel_type_index(_):
     return Index
 
 
-@get_parallel_type.register(object)
-def get_parallel_type_object(o):
-    return Scalar
-
-
 @get_parallel_type.register(_Frame)
 def get_parallel_type_frame(o):
     return get_parallel_type(o._meta)
 
 
-def parallel_types():
-    return tuple(
-        k
-        for k, v in get_parallel_type._lookup.items()
-        if v is not get_parallel_type_object
-    )
+@get_parallel_type.register(object)
+def get_parallel_type_object(_):
+    return Scalar
```

Comment:
I definitley like your solution better.  Do you think there is any need for a `get_parallel_type` test, or do you think the functionality is covered elsewhere?

Reply:
It should be covered in all the `has_parallel_type()` and `new_dd_object()` tests